hypervisor/src/arch/mod.rs

```diff
@@ -11,6 +11,7 @@
 // Copyright © 2020, Microsoft Corporation
 //
 
+#[cfg(all(feature = "mshv"))]
```

Comment:
`#[cfg(feature="mshv")]` should be good enough because there is only one feature specified, right?